pyphare/pyphare/pharein/diagnostics.py

```diff
@@ -145,7 +143,7 @@ def to_dict(self):
 # ------------------------------------------------------------------------------
 
 def population_in_model(population):
-    return population in [p for p in global_vars.sim.model.populations] + ["all","ions"]
+    return population in [p for p in global_vars.sim.model.populations]
```

Comment:
don't remember why we dropped those. I think it was causing a bug somewhere?

Reply:
yeah, I think there was something